python/lzreposync/src/lzreposync/__init__.py

```diff
@@ -0,0 +1,218 @@
+#  pylint: disable=missing-module-docstring
+
+import argparse
+import logging
+
+from lzreposync import db_utils, updates_util
+from lzreposync.db_utils import (
+    get_compatible_arches,
+    get_channel_info_by_label,
+    get_all_arches,
+    create_channel,
+    ChannelAlreadyExistsException,
+)
+from lzreposync.deb_repo import DebRepo
+from lzreposync.import_utils import (
+    import_package_batch,
+    batched,
+    import_repository_packages_in_batch,
+)
+from lzreposync.rpm_repo import RPMRepo
+
+
+def main():
+    parser = argparse.ArgumentParser(
```

Comment:
In the future, i.e. not in this PR, we should align the CLI interface as much as possible with `spacewalk-repo-sync`. We don't have to support all the flags, a subset is fine. Right now we have conflicting flags (e.g. `-d` means _debug_ here but _dry-run_ in `spacewalk-repo-sync`)